lms/djangoapps/django_comment_client/base/views.py

```diff
@@ -187,8 +187,11 @@ def create_thread(request, course_id, commentable_id):
         'title': post["title"],
     }
 
-    if 'context' in post:
-        params['context'] = post['context']
+    # Check for whether this commentable belongs to a team, and add the right context
+    if commentable_team(commentable_id) is not None:
+        params['context'] = 'standalone'
```

Comment:
Could we factor out the valid context values into an enum? I don't want to have these strings littered all over the codebase.